src/utils/ProjectUtils.hx

```diff
@@ -189,18 +189,25 @@ class ProjectUtils
 		}
 		return false;
 	}
-	
+
 	public static function openWithIntelliJIDEA(projectPath:String, projectName:String):Bool
 	{
 		var projectFile = CommandUtils.combine(projectPath, ".idea");
-		
+
 		if (FileSys.exists(projectFile))
 		{
-			if (FileSys.isMac || FileSys.isLinux) // TODO: test on linux
+			if (FileSys.isMac)
+			{
+				Sys.command("open", ["-a",FlxTools.settings.IDEA_Path, projectPath]);
+			}
+			else if (FileSys.isLinux) // TODO: test on linux
```

Comment:
Not tested using flixel and idea. 
As I don't yet have idea on linux, will download and test in idea tomorrow.
I have tested this functionality with other applications. e.g.
On ubuntu I opened a text file in gedit, and used disown to unlink it from the terminal, so I could close the terminal without closing gedit.

I have tested  "& disown" on mac with idea though and it works well. But the recommended way to open apps in mac is "open -a".